client/src/components/Lists/Recipes.tsx

```diff
@@ -0,0 +1,29 @@
+import SmallRecipeDisplay from '../Displays/SmallRecipeDisplay'
+import { useNavigate } from 'react-router-native'
+import React from 'react'
+
+interface RecipesProps {
+  recipes: RecipeObject[];
+  previousLocation: string;
+}
+
+const Recipes = ({ recipes, previousLocation }: RecipesProps) => {
+  const navigate = useNavigate()
+
+  const handlePressFeedPost = (id: number) => {
+    navigate(`/recipe/${id}`, {
```

Comment:
Näis vissii toimi se navigate(-1) suoraa minkä elias oli tonne löytäny, kantsii testaa nii pääsee aika paljo vähemmäl

Reply:
ton pitäs ohjata reseptiin eikä taaksepäin
voisin tosin korjaa ton -1 siihen toiseen kohtaan

Reply:
Aajoo :D Katoin vähä äkäsee